File: java/Samples/BiometricStandards/VB/IIRecordSampleVB/AddIrisForm.vb
Imports System
Imports System.Windows.Forms
Imports Neurotec.Biometrics.Standards

Partial Public Class AddIrisForm
	Inherits Form
	Public Sub New(ByVal version As NVersion)
		InitializeComponent()

		If version = IIRecord.VersionAnsi10 OrElse version = IIRecord.VersionIso10 Then
			cbIrisPosition.Items.Add(System.Enum.GetName(GetType(BdifEyePosition), BdifEyePosition.Left))
			cbIrisPosition.Items.Add(System.Enum.GetName(GetType(BdifEyePosition), BdifEyePosition.Right))
		Else
			cbIrisPosition.Items.AddRange(System.Enum.GetNames(GetType(BdifEyePosition)))
		End If
		cbIrisPosition.SelectedIndex = 0
	End Sub

	Public Property IrisPosition() As BdifEyePosition
		Get
			Return CType(System.Enum.Parse(GetType(BdifEyePosition), cbIrisPosition.SelectedItem.ToString()), BdifEyePosition)
		End Get
		Set(ByVal value As BdifEyePosition)
			cbIrisPosition.SelectedItem = System.Enum.GetName(GetType(BdifEyePosition), value)
		End Set
	End Property
End Class

File: java/Samples/BiometricStandards/CS/IIRecordSampleCS/AddIrisForm.cs
using System;
using System.Windows.Forms;
using Neurotec.Biometrics.Standards;

namespace Neurotec.Samples
{
	public partial class AddIrisForm : Form
	{
		public AddIrisForm(NVersion version)
		{
			InitializeComponent();

			if (version == IIRecord.VersionAnsiCurrent || version == IIRecord.VersionIsoCurrent)
			{
				cbIrisPosition.Items.Add(Enum.GetName(typeof(BdifEyePosition), BdifEyePosition.Left));
				cbIrisPosition.Items.Add(Enum.GetName(typeof(BdifEyePosition), BdifEyePosition.Right));
			}
			else
				cbIrisPosition.Items.AddRange(Enum.GetNames(typeof(BdifEyePosition)));
			cbIrisPosition.SelectedIndex = 0;
		}

		public BdifEyePosition IrisPosition
		{
			get
			{
				return (BdifEyePosition)Enum.Parse(typeof(BdifEyePosition), cbIrisPosition.SelectedItem.ToString());
			}
			set
			{
				cbIrisPosition.SelectedItem = Enum.GetName(typeof(BdifEyePosition), value);
			}
		}
	}
}
